src/implicit.f90

```diff
@@ -742,6 +765,11 @@ subroutine  inttimp (var1,dvar1,npaire,isc,forcing1)
         gg=>ggm211; hh=>hhm211; ss=>ssm211; rr=>rrm211; vv=>vvm211; ww=>wwm211; zz=>zzm211
      elseif ((isc.gt.0).and.(nclyS1.eq.2).and.(nclySn.eq.1).and.(npaire.eq.1)) then
         gg=>ggm211t(:,isc); hh=>hhm211t(:,isc); ss=>ssm211t(:,isc); rr=>rrm211t(:,isc); vv=>vvm211t(:,isc); ww=>wwm211t(:,isc); zz=>zzm211t(:,isc)
+     elseif(present(mhdvar)) then
+          if (nclyB1(mhdvar).eq.2 .and. nclyBn(mhdvar).eq.2) then   
+             gg=>ggmB(:,mhdvar); hh=>hhmB(:,mhdvar); ss=>ssmB(:,mhdvar); rr=>rrmB(:,mhdvar); vv=>vvmB(:,mhdvar); ww=>wwmB(:,mhdvar);
+             zz=>zzmB(:,mhdvar)
```

Comment:
Print an error if `mhdvar` is present and the boundary conditions for the magnetic field are not 2 and 2 ?

Reply:
We have removed the implicit time scheme for a MHD variable since it was not used. If necessary we'll add it back.